streams/ru_okkotv.m3u

```diff
@@ -21,51 +21,51 @@ https://okkotv-live.cdnvideo.ru/channel/Amedia_Hit_HD.m3u8
 http://okkotv-live.cdnvideo.ru/channel/dash/amedia_premium_hd/playlist.mpd
 #EXTINF:-1 tvg-id="",Bolt (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/Bolt_HD.m3u8
-#EXTINF:-1 tvg-id="",Da Vinci (480p) [Geo-blocked]
+#EXTINF:-1 tvg-id="DaVinciRussia.ru",Da Vinci (480p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/dash/da_vinci/playlist.mpd
-#EXTINF:-1 tvg-id="",Duck TV (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id=""ducktv.sk,Duck TV (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/DuckTV.m3u8
-#EXTINF:-1 tvg-id="",KHL (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="KHL.ru",KHL (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/dash/khl_hd_2/playlist.mpd
-#EXTINF:-1 tvg-id="",KHL Prime (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="KHLPrime.ru",KHL Prime (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/dash/khl_prime_hd/playlist.mpd
-#EXTINF:-1 tvg-id="",RTG (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="RTGTV.ru",RTG (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/RTG_HD.m3u8
-#EXTINF:-1 tvg-id="",RU.TV HD (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="RUTV.ru",RU.TV HD (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/RUTV_HD.m3u8
 #EXTINF:-1 tvg-id="",Star Cinema (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/Star_Cinema_HD.m3u8
-#EXTINF:-1 tvg-id="",Star Family (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="StarFamily.ru",Star Family (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/Star_Family_HD.m3u8
-#EXTINF:-1 tvg-id="",START Air (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="STARTAir.ru",START Air (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/START_Air_HD.m3u8
-#EXTINF:-1 tvg-id="",START World (1080p) [Geo-blocked]
+#EXTINF:-1 tvg-id="STARTWorld.ru",START World (1080p) [Geo-blocked]
 https://okkotv-live.cdnvideo.ru/channel/START_World_HD.m3u8
 #EXTINF:-1 tvg-id="",TERRA (1080p) [Geo-blocked]
```

Comment:
```suggestion
#EXTINF:-1 tvg-id="TERRA.ru",TERRA (1080p) [Geo-blocked]
```

Reply:
Not in [database.](https://iptv-org.github.io/?q=TERRA.ru)